vimms/MassSpec.py

```diff
@@ -458,10 +482,11 @@ def _get_all_mz_peaks(self, chemical, query_rt, ms_level, isolation_windows):
         # apply noise if any
         noisy_mz_peaks = []
         for i in range(len(mz_peaks)):
-            original_mz, original_intensity = mz_peaks[i]
+            original_mz = mz_peaks[i][0]
+            original_intensity = mz_peaks[i][1]
             noisy_mz = self.mz_noise.get(original_mz, ms_level)
             noisy_intensity = self.intensity_noise.get(original_intensity, ms_level)
-            noisy_mz_peaks.append((noisy_mz, noisy_intensity))
+            noisy_mz_peaks.append((noisy_mz, noisy_intensity, mz_peaks[i][2], mz_peaks[i][3], mz_peaks[i][4]))
```

Comment:
Is this a good idea? Don't we want the `noisy_mz_peaks` object to return a list of tuples like we get from the real MS?

Reply:
Or does that get trimmed later in the simulator?